src/into_dart.rs

```diff
@@ -43,6 +45,13 @@ impl IntoDart for () {
     }
 }
 
+#[cfg(feature="anyhow")]
+impl IntoDart for anyhow::Error {
+    fn into_dart(self) -> DartCObject {
+        self.to_string().into_dart()
```

Comment:
Hmm I am wondering whether this is sufficient or not? By https://docs.rs/anyhow/1.0.58/anyhow/struct.Error.html, seems that the debug format `{:?}` prints out more information?

Or, if we want more structured solutions we can format it by ourselves